spec/lib/api/redfish-1.0/event-service-spec.js

```diff
@@ -39,62 +39,41 @@ describe('Redfish Event Service', function () {
             }
         }]
     };
-    
-    before('start HTTP server', function () {
-        this.timeout(5000);
+
+    before(function() {
```

Comment:
Can this before be combined with this one https://github.com/RackHD/on-http/pull/697/files#diff-cf0f2855402ebeda8abc6126044353caR54 ?